### ISSUE TITLE
VR videos not downloading correctly!

### ISSUE THREAD

**Issue by randoomx8** [jely2002/youtube-dl-gui#213]
**Describe the bug**
Raw VR videos have a left and right side (or top and bottom for some older ones), however, when downloaded via this app, it instead downloads the _**combined**_ video of left and right (for 180 videos, 360 returns stuff in EAC, which is unusable). The video is still playable, but there is horrible streching and no depth perception.  

**To Reproduce**
1. Go to any youtube video that is marked as VR, ~~180 or~~ 360, ~~does not really mater.~~
2. Download it via the app
3. Try to play it in a video player (I recommend GoVR but it should work on anything)
4. Become sad.

**Expected behavior**
The video is downloaded raw, with both sides intact so that they may be processed by the app itself.

### SOLUTION FOUND:
See comment below https://github.com/jely2002/youtube-dl-gui/issues/213#issuecomment-938168689  
_**Would be nice to see the app implement this natively for 360 videos.**_

**Additional info (please complete the following information):**
 - OS: Win 10
 - Application version 2.3.1
 - Application type installer

**Comment by randoomx8** [comment]
https://github.com/ytdl-org/youtube-dl/issues/22690
https://youtube-eng.googleblog.com/2017/03/improving-vr-videos.html
https://blog.google/products/google-ar-vr/bringing-pixels-front-and-center-vr-video/

It seems like google made it's own format, however, it seems like some apps are capable of converting them back to regular VR.
https://play.google.com/store/apps/details?id=se.chai.vrtv&hl=en
https://www.videoder.com/

**Comment by randoomx8** [comment]
I made a batch file to automate my needs while I wait, here is the code if anyone else wants it:
```
@echo off
Color 1F
echo Welcome to Alemismun's 360 YDL CLI, be sure this program is on the same folder as your ffmpeg and youtube-dl binaries.
echo This CLI can only download 360 videos, 180 videos are believed to be impossible.
echo.
echo Tell your favourite content creators to use better platforms! Such as iwara for MMD VR dance videos.
echo.
echo.
echo [CLI Protip: Right click on this window to paste a link]
set /P videolink="Enter video link... (eg https://www.youtube.com/watch?v=8DqSh_T20WY)"
:VideoSelect
cls
youtube-dl -F %videolink% --user-agent ''
echo.
echo Automatically select best settings?
set /P usebestsetting="Y/N"
if /I "%usebestsetting%" EQU "Y" goto :AutoBest
if /I "%usebestsetting%" EQU "N" goto :ManualQualitySelect
echo.
echo ERROR: INVALID ANSWER.
pause
goto VideoSelect
:ManualQualitySelect
set /P audiosetting="Enter audio quality setting..."
set /P videosetting="Enter video quality setting..."
cls
youtube-dl -f %videosetting%+%audiosetting% --merge-output-format mkv %videolink% --user-agent ''
pause
exit

:AutoBest
youtube-dl -f bestvideo+bestaudio --merge-output-format mkv %videolink% --user-agent ''
pause
exit
````

**Comment by randoomx8** [comment]
Some one noted on the solution post that there were some clarity and resolution issues, I can indeed see them... I will try to find a way around them.

**Comment by jely2002** [comment]
Hi! Thanks a lot for the research you have already done. Would adding an option, to add an empty user agent string work to solve this problem?

**Comment by randoomx8** [comment]
> 
> 
> Hi! Thanks a lot for the research you have already done. Would adding an option, to add an empty user agent string, work to solve this problem?

Probably, I think. Its worth the try!

**Comment by jely2002** [comment]
This feature has been released in 2.4.0 :)

**Comment by yeahman45** [comment]
> ### I CAN CONFIRM THAT THIS DOES WORK, EVEN IF A BIT GLITCHY
> Doing `youtube-dl -f 302+251 --merge-output-format mkv https://www.youtube.com/watch?v=8DqSh_T20WY --user-agent ''` to download a test video works!
> 
> I can't get it to merge with audio (seems to think FFMPEG is not installed, even though it is), but the video is fine!! Please implement an option in your app that lets users download VR videos using the command above. (See [ytdl-org/youtube-dl#15267 (comment)](https://github.com/ytdl-org/youtube-dl/issues/15267#issuecomment-395625496) for more details)

does this still work? does not seem so.
jely2002 also closed this issue as completed but it does not seem to work on his app either.
video tested : https://www.youtube.com/watch?v=JVMQhCKhm5E&t=4s
